chain/indexer/src/streamer/mod.rs

```diff
@@ -249,7 +249,7 @@ pub(crate) async fn start(
 ) {
     info!(target: INDEXER, "Starting Streamer...");
     let mut indexer_db_path = nearcore::get_store_path(&indexer_config.home_dir);
-    indexer_db_path.push_str("/indexer");
+    indexer_db_path.push("/indexer");
```

Comment:
```suggestion
    indexer_db_path.join("indexer");
```

Reply:
Wait, no:

```suggestion
    indexer_db_path.push("indexer");
```

It was odd to see push with a leading slash. Looking at the docs, it's clear why: it'll replace the path instead of pushing onto it.

Reply:
fixed